docs/src/internals/montecarlo.md

```diff
@@ -2,15 +2,11 @@
 
 ## Overview
 
-Sensitivity Analysis support consists of four primary user-facing elements:
+Sensitivity Analysis support consists of two primary user-facing elements:
```

Comment:
> `consists of two primary user-facing elements:` 

Change to `consists of the following user-facing elements:` and avoid having to change this again!